packages/express-api-server/src/express-api-server.js

```diff
@@ -75,12 +77,35 @@ class ExpressApiServer extends Clonable {
     );
     this.app = expressApp.initialize();
 
-    if (port && port > 0) {
-      this.server = this.app.listen(port, () => {
-        const logger = this.container.get('logger');
-        logger.info(`${this.settings.tag} listening on port ${port}!`);
-      });
+    if (!port || port < 1) {
+      return false;
+    }
+
+    let expressServer;
+    const fs = this.container.get('fs');
+    let protocol = '';
+    if (fs.existsSync(this.settings.key) && fs.existsSync(this.settings.cert)) {
```

Comment:
I would first check if the two keys have values before try to use existsSync. becaue if there are values but fies do neot exist i would rather throw an error than silently falling back to HTTP, or?

Reply:
Completely agree with you, I'm changing it

Reply:
maybe dependong on the other discussion (return false vs throw) maybe just return false when invalid confg  (and not throw)